src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
```

Comment:
Link to a stylesheet with .scss extension won't work correctly in a browser. Stylesheets should have a .css extension. You should compile your SCSS to CSS and link to the compiled CSS file instead.